### PR TITLE
Unborks the greentext being too greentextful

### LINKED ISSUE #17697: Greentext item spams round end
http://pastebin.com/wbCJ32is

After asking the unloved rock WHY this occured, he said that it was normal for this to happen

I've really no clue what caused this.

### PR DESCRIPTION
**Pull request by Incoming5643**
Fixes #17697

### CODE REVIEW

File: code/modules/events/wizard/greentext.dm [hunk 1]
```diff
@@ -68,6 +68,7 @@
 		new_holder.mind.objectives += O
 		new_holder.attack_log += "\[[time_stamp()]\] <font color='green'>Won with greentext!!!</font>"
 		color_altered_mobs -= new_holder
+		burn_state == ON_FIRE
```
Comment (RemieRichards) [inline code/modules/events/wizard/greentext.dm]: nice job.

Comment (Incoming5643) [inline code/modules/events/wizard/greentext.dm]: effort consummate with importance

**Comment by KazeEspada** [comment]
Woops!

**Comment by KorPhaeron** [comment]
This is pretty hacky and silly

**Pull request merged**